### PR TITLE
Set-hover-state-for-dropdown-items

### PR DESCRIPTION
**Pull request by adetoye-dev**
Add staking and delegation to menu items

image

### CODE REVIEW

File: src/components/AppLayout/NavHeader.tsx [hunk 1]
```diff
@@ -72,6 +74,8 @@ const NavHeader = ({ className, sidedrawer, setSidedrawer, displayName, isVerifi
 	const dispatch = useDispatch();
 	const { resolvedTheme: theme } = useTheme();
 
+	const delegationLinks: { [key: string]: string } = { kasuma: 'https://kusama.polkassembly.io/delegation', polkadot: 'https://polkadot.polkassembly.io/delegation' };
```
Comment (rajdeep7Singh) [inline src/components/AppLayout/NavHeader.tsx:77]: spelling should be `kusama`

File: src/components/AppLayout/NavHeader.tsx [hunk 2]
```diff
@@ -134,14 +138,48 @@ const NavHeader = ({ className, sidedrawer, setSidedrawer, displayName, isVerifi
...
+					</span>
+				</a>
+			)
+		},
+		{
+			className: 'logo-class',
+			key: 'Delegation',
+			label: (
+				<a
+					href={delegationLinks[network]}
```
Comment (rajdeep7Singh) [inline src/components/AppLayout/NavHeader.tsx:172]: can be changed to `https://${network}.polkassembly.io/delegation`

Comment (rajdeep7Singh) [reply src/components/AppLayout/NavHeader.tsx:172]: This should only work for Polkadot and Kusama. We don't have dashboards for other chains

File: src/components/AppLayout/NavHeader.tsx [hunk 3]
```diff
@@ -117,9 +119,9 @@ const NavHeader = ({ className, sidedrawer, setSidedrawer, displayName, isVerifi
 					href='https://townhallgov.com/'
 					target='_blank'
 					rel='noreferrer'
-					className='custom-link'
+					className='custom-link after:hidden'
```
Comment (KanishkaRajputd) [inline src/components/AppLayout/NavHeader.tsx:122]: why we are using this after:hidden?

File: src/components/AppLayout/NavHeader.tsx [hunk 4]
```diff
@@ -134,23 +136,60 @@ const NavHeader = ({ className, sidedrawer, setSidedrawer, displayName, isVerifi
 					href='https://polkasafe.xyz/'
 					target='_blank'
 					rel='noreferrer'
-					className='custom-link'
+					className='custom-link after:hidden'
```
Comment (KanishkaRajputd) [inline src/components/AppLayout/NavHeader.tsx:139]: >>why we are using this after:hidden?

File: src/components/AppLayout/NavHeader.tsx [hunk 5]
```diff
@@ -134,23 +136,60 @@ const NavHeader = ({ className, sidedrawer, setSidedrawer, displayName, isVerifi
...
+		},
+		{
+			className: 'logo-class',
+			key: 'Staking',
+			label: (
+				<a
+					href='https://staking.polkadot.network/'
+					target='_blank'
+					rel='noreferrer'
+					className='custom-link after:hidden'
```
Comment (KanishkaRajputd) [inline src/components/AppLayout/NavHeader.tsx:156]: why we are using this after:hidden?

File: src/components/AppLayout/NavHeader.tsx [hunk 6]
```diff
@@ -134,23 +136,60 @@ const NavHeader = ({ className, sidedrawer, setSidedrawer, displayName, isVerifi
...
+					<span className='flex items-center gap-x-2 text-sm font-medium text-bodyBlue hover:text-pink_primary dark:text-blue-dark-high dark:hover:text-pink_primary'>
+						<StakeIcon />
+						<span>Staking</span>
+					</span>
+				</a>
+			)
 		}
 	];
 
+	if (['kusama', 'polkadot'].includes(network)) {
```
Comment (KanishkaRajputd) [inline src/components/AppLayout/NavHeader.tsx:167]: create a variable like 
export const delegationSupportedNetworks = [
	AllNetworks.KUSAMA,
	AllNetworks.POLKADOT
];
in Delegation component
and then use that 
also change this in pages/delegation/index.tsx file

File: src/ui-components/NetworkDropdown.tsx [hunk 7]
```diff
@@ -49,7 +49,10 @@ for (const key of Object.keys(network)) {
 						alt='Logo'
 					/>
 				</span>
-				<span className='text-sm font-medium capitalize text-bodyBlue hover:text-pink_primary dark:text-blue-dark-high'> {keyVal == 'hydradx' ? 'HydraDX' : keyVal} </span>
+				<span className='text-sm font-medium capitalize text-bodyBlue hover:text-pink_primary dark:text-blue-dark-high dark:hover:text-pink_primary'>
+					{' '}
```
Comment (KanishkaRajputd) [inline src/ui-components/NetworkDropdown.tsx:53]: can be removed

File: src/ui-components/NetworkDropdown.tsx [hunk 8]
```diff
@@ -49,7 +49,10 @@ for (const key of Object.keys(network)) {
...
+					{keyVal == 'hydradx' ? 'HydraDX' : keyVal}{' '}
```
Comment (KanishkaRajputd) [inline src/ui-components/NetworkDropdown.tsx:54]: same {''} can remove this

**Review by KanishkaRajputd — CHANGES_REQUESTED** [review]

File: src/components/AppLayout/NavHeader.tsx (hunk 3 above)
Comment (adetoye-dev) [reply src/components/AppLayout/NavHeader.tsx:122]: The after element is absolute so the hover does not work on the menu item.

**Pull request merged**